.travis.yml

```diff
@@ -18,3 +21,7 @@ matrix:
       compiler: clang
     - env: PLATFORM=Win64
       compiler: clang
+    - env: PLATFORM=Win32
+      os: osx
+    - env: PLATFORM=Win64
+      os: osx
```

Comment:
Can you exclude one of osx/clang or osx/gcc as well? I don't think we have to test both compilers on all platforms.